src/components/SavedFiltersMenu.vue

```diff
@@ -54,7 +54,7 @@
   }>()
 
   const emit = defineEmits<{
-    (event: 'update:selectedSearchOption', value: SavedSearch | null): void,
+    (event: 'update:selectedSearchOption', value: SavedFlowRunsSearch | null): void,
```

Comment:
This is unrelated but noticed this event type didn't match the prop type.

Reply:
😭